plugins/catalog/src/routes.ts

```diff
@@ -25,6 +25,6 @@ export const rootRoute = createRouteRef({
 });
 export const entityRoute = createRouteRef({
   icon: NoIcon,
-  path: '/catalog/:kind/:optionalNamespaceAndName/',
+  path: '/catalog/:kind/:optionalNamespaceAndName/*',
```

Comment:
Is this really needed? O_o, didn't think you needed exact matches in react-router v6 anymore